LostTreasure/Lang/en.lua

```diff
@@ -1,33 +1,55 @@
+	SI_LOST_TREASURE_PIN_ICON_TT = "Choose your desired icon for pins.",
+	SI_LOST_TREASURE_PIN_SIZE = "Size",
+	SI_LOST_TREASURE_PIN_SIZE_TT = "Choose the size of the pin on player map.",
+	SI_LOST_TREASURE_MARK_OPTION = "Mark options",
+	SI_LOST_TREASURE_MARK_OPTION_TT = "Defines when a pin should be visible on the player map and in the compass.",
+	SI_LOST_TREASURE_MARK_OPTION1_TT = "Shows a pin only after opening a treasure map or a survey report.",
+	SI_LOST_TREASURE_MARK_OPTION2_TT = "Show pins of treasure maps and survey reports that are in your inventory.",
+	SI_LOST_TREASURE_MARK_OPTION3_TT = "Show all pins regardless of whether you have them in your inventory.",
+	SI_LOST_TREASURE_MARK_MAP_MENU_OPTION1 = "on use",
+	SI_LOST_TREASURE_MARK_MAP_MENU_OPTION2 = "all in inventory",
+	SI_LOST_TREASURE_MARK_MAP_MENU_OPTION3 = "all locations",
+	SI_LOST_TREASURE_PIN_LEVEL = "Level on Map",
+	SI_LOST_TREASURE_PIN_LEVEL_TT = "Higher level draws pin over lower level pins. Increase this value, if a pin is obscured by other pins.",
+	SI_LOST_TREASURE_MARKER_DELAY = "Hiding delay",
+	SI_LOST_TREASURE_MARKER_DELAY_TT = zo_strformat("Adds a delay before deleting a marked location on the map. Only works in conjunction with <<1>> or <<2>>. This will not work when you have choosen option <<3>> or open up the map as it refreshes all pins then.", ZO_SELECTED_TEXT:Colorize(GetString(SI_LOST_TREASURE_MARK_MAP_MENU_OPTION1)), ZO_SELECTED_TEXT:Colorize(GetString(SI_LOST_TREASURE_MARK_MAP_MENU_OPTION2)), ZO_SELECTED_TEXT:Colorize(GetString(SI_LOST_TREASURE_MARK_MAP_MENU_OPTION3))),
 
-ZO_CreateStringId ("LTS_MARK_MAP_MENU_OPTION1", "Mark on use")
-ZO_CreateStringId ("LTS_MARK_MAP_MENU_OPTION2", "Mark all in inventory")
-ZO_CreateStringId ("LTS_MARK_MAP_MENU_OPTION3", "Mark all locations")
+	SI_LOST_TREASURE_SHOW_MINIMAP_HEADER = "Mini Map",
+	SI_LOST_TREASURE_SHOW_MINIMAP = "Show Mini Map",
+	SI_LOST_TREASURE_SHOW_MINIMAP_TT = "Show Mini Map, after using a treasure map or survey report.",
+	SI_LOST_TREASURE_SHOW_MINIMAP_SIZE = "Mini Map Size",
+	SI_LOST_TREASURE_SHOW_MINIMAP_DELAY = "Hiding delay",
+	SI_LOST_TREASURE_SHOW_MINIMAP_DELAY_TT = "Delay (in seconds) before hiding Mini Map, after you picked up the treasure or survey.",
 
-ZO_CreateStringId ("LTS_MARKER_DELAY", "BETA : Marker Deletion Delay")
-ZO_CreateStringId ("LTS_MARKER_DELAY_TOOLTIP", "Adds a delay before deleting a marked location on the map. Only works in conjunction with Mark on Use or Mark all in Inventory. This will not work when you open up the map as it refreshes all pins then.")
\ No newline at end of file
+	SI_LOST_TREASURE_NOTIFICATION_MESSAGE = "New unknown data has been found.",
+	SI_LOST_TREASURE_NOTIFICATION_NOTE = "Please share your new amd unknown treasure maps/crafting surveys data by accepting this notification. You need an ESOUI.com account to make this bug report.",
+}
+
+for ident, str in pairs(strings) do
+	ZO_CreateStringId(ident, str)
+	SafeAddVersion(ident, 1)
+end
```

Comment:
Can you explain why we are not using the SafeStrings local function like it is in the non-english files ?

Reply:
because the string is already created, you will just change the stringValue instead overwriting the whole table entry